### PR TITLE
Features/edit a dream

### PR DESCRIPTION
**Pull request by paolabrb**
I have added the "edit a dream" action.

The edit button is in the dream page (es. dreams/1) - it is only showed for the dreams created by the logged in user.

I also updated the views with primary buttons to make them clearer  to see :)

Can somebody review this? Thanks!

### CODE REVIEW

File: app/views/dreams/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--form to edit the dream-->
+
+<div class="container">
+  <h1>Edit your dream</h1>
+
+  <%= simple_form_for(@dream) do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :vibe %>
```
Comment (iferrier) [inline app/views/dreams/edit.html.erb]: We add a dropdown list in the new view that you could add here as well

File: app/views/dreams/new.html.erb [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+<div class="container">
+  <h1>Add a new dream</h1>
+  <%= simple_form_for(@dream) do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :vibe %>
+  <%= f.input :duration %>
+  <%= f.submit 'create dream'%>
+  <% end %>
+</div>
```
Comment (iferrier) [inline app/views/dreams/new.html.erb]: there will be conflict with what we work on with Adiam

File: app/views/dreams/edit.html.erb (hunk 1 above)
Comment (paolabrb) [reply app/views/dreams/edit.html.erb]: I would add it straight away after merging :)

File: app/views/dreams/new.html.erb (hunk 2 above)
Comment (paolabrb) [reply app/views/dreams/new.html.erb]: The conflict is resolved - it's not showing here but it will have the dropdown :D

**Comment by paolabrb** [comment]
I resolved the conflicts :)

**Pull request merged**